scripts/standardise_validate.py

```diff
@@ -27,6 +27,8 @@ def main() -> None:
     parser.add_argument(
         "--end-datetime", dest="end_datetime", help="end datetime in format YYYY-MM-DD", type=valid_date, required=True
     )
+    parser.add_argument("--metadata-file", dest="metadata_file", help="Sidecar metadata file")
+    parser.add_argument("--metadata-type", dest="metadata_type", help="Sidecar metadata file")
```

Comment:
might also want to expand on the help docs for this, could we say what are some valid options?

Reply:
We haven't documented the CLI arguments on this repo. They are undirectly documented in https://github.com/linz/topo-workflows/blob/master/workflows/imagery/README.md where a change needs to be made to use these two new arguments. For the valid options, I think having them as an enum in the workflow will give direction to the user and avoid any typing/spelling issue to match the values in `MetadataType`.